src/status_im/models/browser.cljs

```diff
@@ -153,7 +153,18 @@
   (let [browsers (into {} (map #(vector (:browser-id %) %) all-stored-browsers))]
     {:db (assoc db :browser/browsers browsers)}))
 
-(defn  initialize-dapp-permissions
+(defn initialize-dapp-permissions
   [{:keys [db all-dapp-permissions]}]
   (let [dapp-permissions (into {} (map #(vector (:dapp %) %) all-dapp-permissions))]
     {:db (assoc db :dapps/permissions dapp-permissions)}))
+
+(defn open-chat-from-browser [host cofx]
+  (let [topic (apply str (map #(let [cc (.charCodeAt % 0)]
+                                 (cond (or (and (> cc 96) (< cc 123))
+                                           (and (> cc 64) (< cc 91))
+                                           (and (> cc 47) (< cc 58)))
```

Comment:
what's behind those numbers? can they be extracted to constants or something more descriptive?

Reply:
i'm not sure it's just codes for a-z, A-Z, 0-9

Reply:
Can this code be extracted to a separate function at least with some name which will explain what happens actually? It's not clear what is going on here and why.